### ISSUE TITLE
[BUG] Non existing cfngin-bucket is not created

### ISSUE THREAD

**Issue by tonyfruzza** [onicagroup/runway#1217]
### Bug Description

A deployment of a fresh cfngin project with no prior back end staging bucket failed:

```
[runway] stacks.yaml:init (in progress)
[runway] skipped; cfngin_bucket not defined
[runway] stacks.yaml:init (complete)
[runway] stacks.yaml:deploy (in progress)
Traceback (most recent call last):
  File "runway-script.py", line 2, in <module>
  File "click/core.py", line 1128, in __call__
  File "click/core.py", line 1053, in main
  File "runway/_cli/main.py", line 33, in invoke
  File "click/core.py", line 1659, in invoke
  File "click/core.py", line 1395, in invoke
  File "click/core.py", line 754, in invoke
  File "click/decorators.py", line 26, in new_func
  File "runway/_cli/commands/_deploy.py", line 47, in deploy
  File "runway/core/__init__.py", line 61, in deploy
  File "runway/core/__init__.py", line 260, in __run_action
  File "runway/core/components/_deployment.py", line 313, in run_list
  File "runway/core/components/_deployment.py", line 121, in deploy
  File "runway/core/components/_deployment.py", line 273, in __sync
  File "runway/core/components/_deployment.py", line 188, in run
  File "runway/core/components/_module.py", line 325, in run_list
  File "runway/core/components/_module.py", line 176, in deploy
  File "runway/core/components/_module.py", line 242, in run
  File "runway/module/cloudformation.py", line 68, in deploy
  File "runway/cfngin/cfngin.py", line 132, in deploy
  File "runway/cfngin/actions/base.py", line 169, in execute
  File "runway/cfngin/actions/deploy.py", line 563, in pre_run
  File "runway/cfngin/actions/base.py", line 164, in ensure_cfn_bucket
runway.cfngin.exceptions.CfnginBucketNotFound: cfngin_bucket does not exist best-demo-us-east-1
````

### Expected Behavior

When bootstrapping a new Cfngin project that doesn't have a S3 bucket backend staging area I'd expect that the unique bucket name defined for `cfngin_bucket` would be created by a deploy.

### Steps To Reproduce

```
runway new
runway gen-sample cfn
```

Make the minimal changes to the .env and remove the second module within `runway.yml`. Run this and you'll run into the same issue.

### Runway version

2.5.0

### Installation Type

direct download (curl, wget, etc)

### OS / Environment

- OS: MacOS 12.1
- Python 3.10.1

### Anything else?

This issue is not present with the same exact configuration within 2.4.4 (latest 2.4.x release)

**Comment by ITProKyle** [comment]
Thanks, I'll look into this.

**Comment by ITProKyle** [comment]
This looks to have been caused by the change to `__bool__` in https://github.com/onicagroup/runway/pull/1133 causing this condition to unintentionally be met: https://github.com/onicagroup/runway/blob/v2.5.0/runway/cfngin/actions/init.py#L109-L111

**Comment by ITProKyle** [comment]
The patch for this should be released early next week (likely Monday) in 2.6.0. It has taken a bit longer to get out than I would have liked due to a set of minor changes that were in progress prior to the patch.
